releasenotes/notes/add-sparse-pauli-conversion-methods-0eee4738fd1ad5f0.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Adds methods :meth:`.QubitSparsePauli.to_pauli`, :meth:`.PhasedQubitSparsePauli.to_pauli`,
+    :meth:`.QubitSparsePauliList.to_pauli_list`, and
+    :meth:`.PhasedQubitSparsePauliList.to_pauli_list` conversion methods. These methods convert the
+    sparse objects into the corresponding dense versions, :class:`Pauli` and :class:`PauliList`
```

Comment:
the links to Pauli and PauliList didn't work for me.

Reply:
I've added `~` to the references... not sure if that will fix it. The docs build is failing for me locally. Will try to get it to work but for now hopefully that fixes it?